CHANGELOG.md

```diff
@@ -4,6 +4,19 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Update Razor to 9.0.0-preview.24467.1
```

Comment:
Can you move this to a new `# Latest Section` and rename the existing latest to 2.48.x?